tests/contrib/molten/test_molten.py

```diff
@@ -55,6 +55,12 @@ def test_route_success(self):
         self.assertEqual(span.get_tag('http.url'), '/hello/Jim/24')
         self.assertEqual(span.get_tag('http.status_code'), '200')
 
+        # See test_resources below for specifics of this difference
+        if MOLTEN_VERSION >= (0, 7, 2):
+            self.assertEqual(len(spans), 18)
```

Comment:
Thank you, I know it probably seems a little silly, but a super easy sanity check, at least until we are using `self.assert_structure()` in all tests.